airflow/providers/google/cloud/utils/credentials_provider.py

```diff
@@ -311,9 +315,20 @@ def _get_credentials_using_key_secret_name(self):
         project_id = credentials.project_id
         return credentials, project_id
 
+    def _get_credentials_using_credential_config_file(self):
```

Comment:
can we add unit tests for this?

Reply:
Yes, with the caveat that it's not going to test much.